[GEfdTyrb] - Телефон

KossaDenis opened the pull request:
- Добавил секцию с телефоном

KossaDenis committed:
правки с иконкой и стилями

Commit:
price-card component

Commit:
[O5fNSvsN] Цены
-- добавлен price-card компонент

KossaDenis committed:
Экран телефон

KossaDenis committed:
Экран телефон

KossaDenis committed:
[GEfdTyrb] - Телефон
- Создать экран с телефоном

KossaDenis committed:
[GEfdTyrb] - Телефон
- Создать экран с телефоном

weekkkk committed:
create scroll

weekkkk committed:
create scroll

KossaDenis committed:
[GEfdTyrb] - Телефон
- Создать экран с телефоном

Commit:
[O5fNSvsN] Цены
- добавил компонент карточки для прайс листа
- реализовал фичу переключения списка прайс листа через таб

Commit:
[O5fNSvsN] Цены
- добавил компонент карточки для прайс листа
- реализовал фичу переключения списка прайс листа через таб

Commit:
Merge branch 'feature/O5fNSvsN' of https://github.com/weekkkk/x-install--landing-page into feature/O5fNSvsN

KossaDenis committed:
[GEfdTyrb] - Телефон
- Создать секцию с телефоном

KossaDenis committed:
[GEfdTyrb] - Телефон
- Создать секцию с телефоном

Commit:
[O5fNSvsN] Цены
-- добавлены градиентные фигуры по бокам блока "Цены"

KossaDenis committed:
[GEfdTyrb] - Телефон
- Создать секцию с телефоном

KossaDenis committed:
[GEfdTyrb] - Телефон
- Создать секцию с телефоном

KossaDenis committed:
[qr0RWWHT] - Цели

KossaDenis committed:
[qr0RWWHT] - Цели

KossaDenis committed:
[GEfdTyrb] - Телефон
- Создать секцию с телефоном

KossaDenis committed:
[GEfdTyrb] - Телефон
- Создать секцию с телефоном

weekkkk commented on src/app/assets/css/tailwind.css:
```diff
@@ -10,6 +10,14 @@
+  .moving-element {
+    /* transition: transform 0.1s linear; */
+    /* will-change: transform; */
+  }
```
убери

weekkkk commented on src/pages/main/ui/page/index.vue:
```diff
@@ -1,10 +1,41 @@
+  onUnmounted(() => {
+    document.removeEventListener("scroll", handleScroll); // Очищаем слушатель при удалении компонента
+  });
+});
```
`onUnmounted` нельзя размещать внутри `onMounted`, когда из GPT копируешь хоть анализируй что он тебе выдает

weekkkk commented on src/pages/main/ui/page/index.vue:
```diff
@@ -1,10 +1,41 @@
+      ticking = true;
+      requestAnimationFrame(updatePositions); // Запускаем один раз за кадр
+    }
+  };
```
калбэки тоже за onMounted нужно выносить

Commit:
[O5fNSvsN] Цены
-- добавлен экспорт UiPriceCardProps в shared/ui
-- все статичные виджета PriceList данные были пересены в price/ui/list/consts/..
-- добавлен ключ для перебора UiPriceCard

KossaDenis committed:
[qr0RWWHT] - Цель
- Создать секцию целей

KossaDenis committed:
[qr0RWWHT] - Цель
- Создать секцию целей

weekkkk commented on src/shared/ui/BlurCircle/index.vue:
в названии папки использую kebab-case

weekkkk commented on src/app/routes/index.vue:
```diff
@@ -1,8 +1,9 @@
 <template>
-  <div class="ml-10 mr-8 my-[2.35rem]">
-    <MainPage />
+  <div class="relative w-full overflow-hidden">
```
зачем этот див

KossaDenis replied:
он обрезает то что выходит за его пределы, сейчас если убрать то появится блок для скролла нижнего (без скролла)

weekkkk commented on src/pages/main/ui/page/index.vue:
```diff
@@ -1,10 +1,41 @@
 <template>
   <LayoutHeaderWidget />
   <MainPageSectionsWelcome class="mt-[10.65rem] h-[52.55rem]" />
+  <MainPageSectionsPhone class="overflow-hidden" />
```
зачем этот класс

weekkkk commented on src/pages/main/ui/page/sections/phone/index.vue:
```diff
@@ -0,0 +1,47 @@
+<template>
+  <div class="h-[46.3rem] overflow-hidden">
```
тут просто w-full напиши

weekkkk committed:
Merge pull request #5 from weekkkk/feature/O5fNSvsN

[O5fNSvsN] Цены

KossaDenis committed:
[MSbAzkYT] - Услуги
- Создать страницу Услуги

KossaDenis committed:
[qr0RWWHT] - Цель
- Создать секцию целей

KossaDenis committed:
[qr0RWWHT] - Цель
- Создать секцию целей

KossaDenis committed:
[MSbAzkYT] - Услуги

KossaDenis committed:
[qr0RWWHT] - Цели
- Исправления + адаптив

KossaDenis committed:
[qr0RWWHT] - Цели
- Исправления + адаптив

Commit:
[O5fNSvsN] Цены
-- адаптив компонента и виджета

KossaDenis committed:
[moN3tQFA] - Шапка
- Адапатив

Commit:
[O5fNSvsN] Цены
-- правки по функционалу

KossaDenis committed:
[moN3tQFA] - Шапка
- Адаптив

KossaDenis committed:
[qr0RWWHT] - Цели
- Правки

KossaDenis committed:
[qr0RWWHT] - Цели
- Правки

KossaDenis reviewed (commented):
добавил адаптив

Commit:
[O5fNSvsN] Цены
-- адаптив для компонента PriceCard
-- правки по фунционалу в логике отрисовки под мобилки,
добавлен один необязательный props для PriceCard
md: {guarantees?: string[]}

KossaDenis committed:
[96r4FJFZ] - Страница приветствия
- Адаптив

KossaDenis committed:
[96r4FJFZ] - Страница приветствия
- Адаптив

KossaDenis committed:
[moN3tQFA] - Шапка
- Добавлено появление хэдэра при скролле вверх

KossaDenis committed:
[GEfdTyrb] - Телефон
- Добавил адаптив

KossaDenis committed:
[GEfdTyrb] - Телефон
- Добавил адаптив

KossaDenis committed:
[GEfdTyrb] - Телефон
- Добавил адаптив

KossaDenis committed:
[qr0RWWHT] - Цель

KossaDenis committed:
[qr0RWWHT] - Цель

KossaDenis committed:
[qr0RWWHT] - Цель
- Правки

KossaDenis committed:
[qr0RWWHT] - Цель
- Правки

KossaDenis committed:
[MSbAzkYT] - Услуги
- Создал секцию услуг

weekkkk committed:
Merge pull request #10 from weekkkk/feature/O5fNSvsN

[O5fNSvsN] Цены

weekkkk committed:
Merge pull request #9 from weekkkk/feature/96r4FJFZ

[96r4FJFZ] - Страница приветствия

KossaDenis committed:
Merge branch 'develop' into feature/moN3tQFA

weekkkk committed:
Merge pull request #8 from weekkkk/feature/moN3tQFA

[moN3tQFA] - Шапка

weekkkk committed:
Merge pull request #7 from weekkkk/feature/qr0RWWHT

[qr0RWWHT] - Цели

KossaDenis committed:
Merge branch 'develop' into feature/1PEbk5IL

weekkkk committed:
Merge pull request #12 from weekkkk/feature/1PEbk5IL

[1PEbk5IL] - Контактная форма

KossaDenis committed:
Merge branch 'develop' into feature/MSbAzkYT

KossaDenis committed:
[S6VizWsU] - Контактная форма
- Создать контактную форму

KossaDenis reviewed (commented):
правки

weekkkk committed:
Merge pull request #11 from weekkkk/feature/MSbAzkYT

[MSbAzkYT] - Услуги

KossaDenis committed:
[moN3tQFA] - Шапка
- Правки

KossaDenis committed:
[moN3tQFA] - Шапка
- Правки

KossaDenis committed:
[]

KossaDenis committed:
[96r4FJFZ] - Страница приветствия
- Правки

KossaDenis committed:
[qr0RWWHT] - Цели
- Правки

KossaDenis committed:
[1PEbk5IL] - Контактная форма
- Правки

KossaDenis committed:
[MSbAzkYT] - Услуги
- Правки

KossaDenis committed:
[GEfdTyrb] - Телефон
- правки

KossaDenis committed:
[GEfdTyrb] - Телефон
- правки

KossaDenis committed:
[GEfdTyrb] - Телефон
- Правки

KossaDenis committed:
Merge branch 'feature/GEfdTyrb' of https://github.com/weekkkk/x-install--landing-page into feature/GEfdTyrb

KossaDenis committed:
[GEfdTyrb] - Телефон
- Правки
